### PR TITLE
Add pushImageTag functionality

### PR DESCRIPTION
**Pull request by adowner**
for software houses that create a lot of images, the maven plugin will try to push all tags when you specify pushImage. I have added a more focused pushImageTag so that only images, specified in the pom, are uploaded to the registry.
This should resolve issue #40

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,6 +94,41 @@ To push the image you just built to the registry, specify the `pushImage` flag.
...
+          <artifactId>docker-maven-plugin</artifactId>
+          <version>VERSION GOES HERE</version>
+          <configuration>
+            <imageName>example</imageName>
+            <baseImage>java</baseImage>
+            <imageTags>
+               <imageTag>${project.version}</imageTag>
+               <imageTag>latest</imageTag>
+            </imageTags>
+            <entryPoint>["java", "-jar", "/${project.build.finalName}.jar"]</entryPoint>
```
Comment (mattnworb) [inline README.md]: I would suggest removing the part of this snippet unrelated to the `<imageTag>` functionality

File: pom.xml [hunk 2]
```diff
@@ -296,7 +296,14 @@
           <findbugsXmlOutputDirectory>${project.build.directory}/findbugs
           </findbugsXmlOutputDirectory>
           <excludeFilterFile>${project.basedir}/findbugs-exclude.xml</excludeFilterFile>
-        </configuration>
+  </configuration>
+        <dependencies>
+	  <dependency>
+	    <groupId>org.apache.ant</groupId>
+            <artifactId>ant</artifactId>
+            <version>1.9.4</version>
```
Comment (mattnworb) [inline pom.xml]: why is this needed?

**Comment by mattnworb** [comment]
Thanks for the contribution @adowner! It looks good to me besides from the parts above, if you can resolve the merge conflict I'd be happy to merge.

Regarding the example configuration, it might be easier to suggest that the tags-to-be-pushed can also be specified with 

    mvn ... docker:build -DpushImageTags -DdockerImageTag=latest -DdockerImageTag=another-tag

File: pom.xml (hunk 2 above)
Comment (adowner) [inline pom.xml]: This is ant findbugs fix from:
https://github.com/spotify/docker-client/pull/171
This affected me

**Comment by adowner** [comment]
updates done, conflicts resolved, thanks for your time.

**Comment by brian-villanueva** [comment]
@mattnworb Any anticipated timeframe for this? Definitely anxious to see this feature incorporated.

**Comment by munglerizer** [comment]
@davidxia is there anything else stopping this PR from being merged? I'd love to be able to move off of my patched version.

**Comment by mattnworb** [comment]
Sorry for the delay!

**Comment by davidxia** [comment]
0.4.0 has been released with this PR
